transformations/diverse_paraphrase/transformation.py

```diff
@@ -0,0 +1,151 @@
+import random
+import numpy as np
+import torch
+from random import sample
+from transformers import FSMTForConditionalGeneration, FSMTTokenizer
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+from transformations.diverse_paraphrase.submod.submodopt import SubmodularOpt
+from transformations.diverse_paraphrase.submod.submodular_funcs import trigger_dips
+
+
+class DiverseParaphrase(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+    heavy = True
+
+    def __init__(self, augmenter="dips", num_outputs=4):
```

Comment:
You should pass seed here too.

Reply:
Should I move the seeds from __init__ to class variables?

Reply:
As an argument to `__init__`?

Reply:
Yes!